hrm-service/src/case/case-routes-v0.ts

```diff
@@ -39,11 +39,12 @@ const casesRouter = SafeRouter();
 casesRouter.get('/', publicEndpoint, async (req, res) => {
   const { accountSid } = req;
   const { sortDirection, sortBy, limit, offset, ...search } = req.query;
+  const { canOnlyViewOwnCases } = req.searchPermissions;
   const cases = await caseApi.searchCases(
     accountSid,
     { sortDirection, sortBy, limit, offset },
     { filters: { includeOrphans: false }, ...search },
-    { can: req.can, user: req.user },
+    { can: req.can, user: req.user, canOnlyViewOwnCases },
```

Comment:
Does it make sense to pass the entire `req.searchPermissions` object through to the business logic layer (similar to the way we pas the entire `req.user` object) so that the method signatures don't change quite so much as we end up adding more permissions in the future?